modules/installation-guide/partials/con_persistent-volume-configuration.adoc

```diff
@@ -42,6 +42,8 @@ If there are several nodes, you can use the `common` strategy, but:
 
 The `common` PVC strategy is not suitable for large multi-node clusters. Therefore, it is best to use it in single-node clusters. However, in combination with the `per-workspace` {orch-namespace} strategy, the `common` PVC strategy is usable for clusters with not more than 75 nodes. The PVC used with this strategy must be large enough to accommodate all projects to prevent a situation in which one project depletes the resources of others.
 
+NOTE: The common PVC will be removed once all of the user's workspaces are deleted. The PVC will be re-created once a new non-ephemeral workspace is started.
+
```

Comment:
I think it will be better to add the note to the section above - https://github.com/eclipse/che-docs/pull/2015/files#diff-942e60eb24c029b670effc74989aadf810f89976974b39ba0ed8148d9054c222R31 just after `
When a workspace is deleted, a corresponding subdirectory (`$\{ws-id}`) is deleted in the PV directory.<br class="Apple-interchange-newline">`

Reply:
I agree, fixed